docs/book/src/clusterctl/developers.md

```diff
@@ -37,7 +37,6 @@ In order to build artifacts for the CAPI core provider, the kubeadm bootstrap pr
 
 ```
 make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api
-make -C test/infrastructure/docker generate-manifests REGISTRY=gcr.io/k8s-staging-cluster-api
```

Comment:
This is usually required, assuming most of the CAPI developers use docker for testing locally.
What about adding a note instead of deleting

Reply:
The immediate  following 3 lines are about build docker provider. Do you  think the notes is necessary here? 
https://github.com/kubernetes-sigs/cluster-api/pull/4220/files#diff-7d6aeba78cbb198ada9c37bb5c9deaf40948598ff8df831955c6118093e0fa63L43-L48

Reply:
Sorry I did not check the rest of the doc.
However, I'm still confused by this sequence, because I would expect to build manifests & build docker images for both CAPI & CAPD, instead, it seems that we are missing build manifests for CAPI...
🤔

Reply:
What would be right sequence? I'm quite new to this so not really knew what it should be. I deleted this line since it looks duplicated to me with the following instructions.

> it seems that we are missing build manifests for CAPI...
Are you saying even *before* the change we missing "build manifest for CAPI"  or you are saying if we delete this line we will be missing that?

Could it be related with the issue that I see different cert-manger being used as discussed [here](https://github.com/kubernetes-sigs/cluster-api/issues/4222#issuecomment-784528462)?

Reply:
@fabriziopandini  let me know you want to proceed with this? should I leave it or delete it. I feel the CAPI manifest are somehow build automatically or implicitly during the process. At least, I didn't experience a problem that during my setup.

I also updated the doc to include my findings, such as https://github.com/kubernetes-sigs/cluster-api/issues/4223.

Reply:
I added 'export PULL_POLICY=IfNotPresent' and hoped the generated manifest will not use AlwaysPull but seems it's not the case. I will revert this line of change for now.

Reply:
correction. `PULL_POLICY=IfNotPresent' worked. I forgot the run the hack scripts after the build :)
I also correct the steps that missing build manifest for capi.